easybuild/tools/github.py

```diff
@@ -721,14 +753,19 @@ def new_pr(paths, title=None, descr=None, commit_msg=None):
     class_label = ','.join([tc for (cnt, tc) in classes_counted if cnt == classes_counted[-1][0]])
 
     if title is None:
-        # mention software name/version in PR title (only first 3)
-        names_and_versions = ["%s v%s" % (ec.name, ec.version) for ec in file_info['ecs']]
-        if len(names_and_versions) <= 3:
-            main_title = ', '.join(names_and_versions)
-        else:
-            main_title = ', '.join(names_and_versions[:3] + ['...'])
+        if file_info['ecs'] and any(file_info['new']):
```

Comment:
@Caylo this should be `all(file_info['new'])`, no? We only want to auto-guess the title if all easyconfigs are new?